lib/Alchemy/Phrasea/Utilities/StringHelper.php

```diff
@@ -0,0 +1,42 @@
+<?php
+
+/*
+ * This file is part of Phraseanet
+ *
+ * (c) 2005-2015 Alchemy
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Alchemy\Phrasea\Utilities;
+
+/**
+ * Class CrLfNormalizer
+ * @package Alchemy\Phrasea\Utilities\String
+ */
+class StringHelper
+{
+    /**
+     * @param string $str
+     * @return string
+     */
+    public static function crlfNormalize($str)
+    {
+        return preg_replace("/\r\n|\r|\n/", "\n", $str);
```

Comment:
m modifier not necessary in this case since no '^' or '$' in exp. Anyway str_replace is better since there is no side-effects between each search/replace.
done.